dashboard/src/api/hooks/useGetProject.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query';
+import { Project } from '../../interfaces/projects';
+import client from '../client';
+import '../mocks/useGetProject.mock';
+
+const getProject = async (projectId?: number) => {
```

Comment:
we should not set the optional

Reply:
we are having these to not have crutches inside component like another state that'll contain contain data and useEffect on top of that